projects/dino/dataset/transform/processing.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.transforms import RandomApply  # noqa: E501
+from mmcv.transforms import BaseTransform, Compose, RandomFlip, RandomGrayscale
+
+from mmselfsup.datasets.transforms import (ColorJitter, RandomGaussianBlur,
+                                           RandomResizedCrop, RandomSolarize)
+from mmselfsup.registry import TRANSFORMS
+
+
+@TRANSFORMS.register_module()
+class DINOMultiCrop(BaseTransform):
```

Comment:
Since this is the project folder, there is no need to require community developer to refactor their code to existing format of MMSelfSup. This is why I create a new transform here.